mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.module.scss

```diff
@@ -1,12 +1,9 @@
 @import "src/styles/variables";
 
-.pageWrapper {
-  max-width: $maxWidthContainer;
-  align-items: center;
-  padding: 0 $gapMedium;
-  margin: $marginMedium auto;
-  gap: $gapBig;
-}
+// Height of header. Need for add background under the header
+$mainImageMarginTop: -103px;
```

Comment:
better to handle it consystently for all similar pages. It will be impossible in future wheb we decided to uncrease header heigh for 5 px for example check it everywhere

Reply:
Fixed